packages/fxa-payments-server/src/index.js

```diff
@@ -1,14 +1,37 @@
-//eslint-disable-next-line no-unused-vars
 import React from 'react';
-import ReactDOM from 'react-dom';
+import { render } from 'react-dom';
+import { createAppStore, actions } from './store';
+import { parseParams } from './utils';
+
+import config from './config';
 import './styles/index.css';
-//eslint-disable-next-line no-unused-vars
-import Payments from './Payments';
-import * as serviceWorker from './serviceWorker';
+import App from './components/App';
+
+function init() {
+  const store = createAppStore();
+  const { dispatch } = store;
+
+  const updateFromParams = () => {
+    const {
+      productId,
+      accessToken
+    } = parseParams(window.location.hash);
```

Comment:
I feel like it might be nice to erase location.hash (or at least accessToken) after reading it, so the URL doesn't look quite so scary (I'm not sure if we should set location.hash or call replaceState). Also if someone reloads or revisits the page, we want them to go through some redirect dance, ideally instead of getting an invalid accessToken and then having it fail in some weird way. (I guess that's an argument for replaceState.)